docs/source/queries/schema-agreement.md

```diff
@@ -1,25 +1,14 @@
 # Schema agreement
```

Comment:
This was a problem before your PR, but this documentation page is missing any mention about automatic schema agreement after schema-changing queries. 

(cc: @muzarski - what we talked about today: the driver does automatic schema agreement after such queries, no need to do it youself)